README.md

```diff
@@ -18,10 +18,12 @@ Assembly:
 * One resistor is connected to rows 40 and 45.
 * One jumper wire is connected to row 40 and GND.
 
```

Comment:
Can you include either a link to the PDF data sheet or include the actual data sheet in a folder called docs within this repo?

Reply:
Added the docs folder